src/components/Player/YoutubePlayer.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react';
+import YouTube, { Options } from 'react-youtube';
+
+type Props = {
+  videoId: string;
+  handleNext: () => void;
+  handleBefore: () => void;
+};
+
+export const YoutubePlayer: React.FC<Props> = ({ videoId, handleNext, handleBefore }) => {
+  const [isPaused, setIsPaused] = useState(false);
+  const [volume, setVolume] = useState(0);
+  const [player, setPlayer] = useState<YT.Player>();
+
+  useEffect(() => {
+    isPaused ? player?.pauseVideo() : player?.playVideo();
+  }, [isPaused, player]);
+  useEffect(() => {
+    player?.setVolume(volume);
+  }, [volume, player]);
+
+  const handlePlaybuttonClick = (): void => {
+    setIsPaused(!isPaused);
+  };
+  const handleMuteButtonClick = (): void => {
+    setVolume(volume === 0 ? 100 : 0);
+  };
+  const handleOnReady = (event: YT.PlayerEvent): void => {
+    setPlayer(event.target);
+  };
+
+  const opts: Options = {
```

Comment:
```suggestion
  const youtubeOptions: Options = {
```

우리 코드에서는 줄이지 말자! (유튜브 api에서는 줄여져있더라도)